python/ray/serve/config.py

```diff
@@ -129,6 +130,10 @@ class DeploymentConfig(BaseModel):
     # the deploymnent use.
     deployment_language: Any = DeploymentLanguage.PYTHON
 
+    name: str
+    version: Optional[str]
```

Comment:
I think we can remove `prev_version`. Was there a verdict on `version`? @simon-mo @edoakes

Reply:
In order to quickly support Java user scenarios in this PR, no `version` and `prev_version` related changes are made. Can you make relevant adjustments later? @simon-mo